src/js/common/components/Style/DesignTokenColors.js

```diff
@@ -1,9 +1,13 @@
 const primitiveColorNames = {
   blue50: '#E6F3FF',
+  blueUI50: '#E2F2FD',
+  blueUI900: '0943A3',
```

Comment:
I will fix this, but it should have included "#" at the start. Otherwise approved! 👍